fundamentals/02.3_aligning_data_objects.ipynb

```diff
@@ -134,16 +466,32 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "arr1_al, arr2_al = xr.align(arr1, arr2, join=\"inner\")\n",
-    "arr2_al"
+    "arr1 = xr.DataArray(\n",
+    "    np.arange(12).reshape(3, 4),\n",
+    "    dims=(\"x\", \"y\"),\n",
+    "    coords={\"x\": [\"a\", \"b\", \"c\"], \"y\": [0, 1, 2, 3]},\n",
+    ")\n",
+    "arr2 = xr.DataArray(\n",
+    "    [0, 1],\n",
+    "    dims=\"x\",\n",
+    "    coords={\"x\": [\"b\", \"d\"]},\n",
+    ")\n",
+    "arr1 - arr2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We see that `arr2` has been broadcasted to 2D  (so dimension \"y\" has been inserted) and the two arrays have been  aligned using `join=\"inner\"` prior to subtraction."
    ]
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "Python [conda env:xarray-tutorial]",
    "language": "python",
-   "name": "python3"
+   "name": "conda-env-xarray-tutorial-py"
```

Comment:
CI assumes `"name": "python3"` Need to fix the CI to actually fail when the book building fails (`jupyter_client.kernelspec.NoSuchKernel: No such kernel named conda-env-xarray-tutorial-py`)

Reply:
Does the manual installation of the kernel (prior the notebook execution) help with this issue?

```bash
python3 -m ipykernel install --user --name conda-env-xarray-tutorial-py
```

Reply:
It'd be nice to have a pre-commit hook to rewrite kernel names =)

Reply:
nbstripout can be configured to strip the whole kernelspec, so that it will default to `python3`. Would that work?

Reply:
That sounds great!